doc/user_guide/user_guide.md

```diff
@@ -172,13 +147,13 @@ mvn package -DskipTests=true
 The assembled jar file with the `-assembly` suffix should be located in the `target/`
 folder.
 
-If you want different version of Spark, you can use profiles `-Pspark3.1` or
-`-Pspark2.4` for Spark `3.1` or `2.4` versions respectively.
+If you want different version of Spark, you can use profiles `-Pspark3.2` or
+`-Pspark3.1` for Spark `3.2` or `3.1` versions respectively.
```

Comment:
```suggestion
`-Pspark3.1` for Spark `3.2` or `3.1` versions, respectively.
```
See https://linguaholic.com/linguablog/comma-before-respectively/

Reply:
Nice, didn't know it :))